webui/react/src/components/Table.tsx

```diff
@@ -25,21 +29,19 @@ type TaskRenderer = (text: string, record: CommandTask, index: number) => React.
 
 /* Table Column Renderers */
 
-export const actionsRenderer: Renderer = (_, record) => {
-  if (isExperiment(record)) {
-    return <TaskActionDropdown task={experimentToTask(record)} />;
-  } else {
-    return <TaskActionDropdown task={record as CommandTask} />;
-  }
+export const durationRenderer = (times: StartEndTimes): React.ReactNode => {
+  return shortEnglishHumannizer(getDuration(times));
 };
 
-export const relativeTimeRenderer = (date: Date): React.ReactNode => (
-  <Tooltip title={date.toLocaleString()}>
-    <TimeAgo datetime={date} />
-  </Tooltip>
-);
+export const relativeTimeRenderer = (date: Date): React.ReactNode => {
+  return (
```

Comment:
question: why the style change?

Reply:
didn't mean to, I think I had additional code here added at one point then stripped it all out. So it should look like how it was before.